src/androidTest/java/com/couchbase/lite/DatabaseAttachmentTest.java

```diff
@@ -168,6 +173,28 @@ public void testAttachments() throws Exception {
     }
 
     /**
+     * - (void) test13_GarbageCollectAttachments
+     */
+    public void test13_GarbageCollectAttachments() throws CouchbaseLiteException{
```

Comment:
In many times, I would like to compare the test code with iOS code. Without same name, it is harder to find test. Personally I'd like to use same name.